hashes/src/sha256t.rs

```diff
@@ -43,7 +43,7 @@ pub struct Hash<T: Tag>(
 );
 
 impl<T: Tag> Hash<T> {
-    fn internal_new(arr: [u8; 32]) -> Self { Hash(arr, Default::default()) }
+    fn internal_new(arr: [u8; 32]) -> Self { Hash(arr, PhantomData::default()) }
```

Comment:
If you're going to spell `PhantomData` you might as well drop `::default()` part.

Reply:
ha ha, good point. I"m not 100% sure this is an improvement but it does save one glancing up a few lines to check the type created by `Default::default()`.